src/routes/session/session-get.ts

```diff
@@ -1,18 +1,41 @@
-import { NextFunction, Request, Response } from "express";
+import { Request, Response } from "express";
 import { createUrlWithQueryString } from "utils/create-url-with-query-string";
 
-export const SessionGet = (req: Request, res: Response, next: NextFunction) => {
-	if (!req.params[0]) {
-		return next(new Error("Missing redirect url for session.  Needs to be in format `/session/:redirectUrl`"));
-	}
-
-	const url = createUrlWithQueryString(req, req.params[0]);
+export const SessionGet = (req: Request, res: Response) => {
+	const url = createUrlWithQueryString(req, req.params[0] || "");
+	const { title, loadAutoAppCreation } = configForPage(req);
 
-	return res.render("session.hbs", {
-		title: "Logging you into GitHub",
-		APP_URL: process.env.APP_URL,
+	res.render("session.hbs", {
 		redirectUrl: new URL(url, process.env.APP_URL).href,
```

Comment:
What happens here when req.params[0] isn't defined and we set url to an empty string? Is there a reason why you removed the exception here? I think we should still include this.

Reply:
If there is no `req.params[0]` then empty string is used as url. This condition is done in line 5. 
There are 1 test case for this case too `Testing the route just `session` when no query params`